data/battle_scripts/subscripts/subscript_0012_UPDATE_STAT_STAGE.s

```diff
@@ -14,6 +14,7 @@ _010:
     Wait 
 
 _018:
+    CompareVarToValue OPCODE_FLAG_NOT, BSCRIPT_VAR_BATTLE_STATUS, BATTLE_STATUS_MOVE_SUCCESSFUL, _023
```

Comment:
oh shucks, was rapid spin always raising speed?  that's funny
should really do rapid spin properly at some point 😂